drivers/f90/driver.f90

```diff
@@ -585,7 +583,12 @@ PROGRAM DRIVER
                DO i=1, nat, 3
                   dip = dip -1.1128d0 * atoms(i,:) + 0.5564d0 * (atoms(i+1,:) + atoms(i+2,:))
                ENDDO
-               ! do not compute the virial term
+            ELSEIF (vstyle == 27) THEN ! short-range qtip4pf potential.
+               IF (mod(nat,3)/=0) THEN
+                  WRITE(*,*) " Expecting water molecules O H H O H H O H H but got ", nat, "atoms"
+                  STOP "ENDED"
+               ENDIF
+               CALL qtip4pf_sr(atoms,nat,forces,pot,virial)
```

Comment:
is this tested for not orthorhombic cells? I understand that the Ewald was the main problem but if it is not tested I will play on the safe side and leave as qtip4pf (lr)

Reply:
(i.e. abort if the cell is not orthorhombic)

Reply:
yes exactly. this is why we use pbc="false" in the ffsocket.